dali/base/dadfs.cpp

```diff
@@ -5468,7 +5478,10 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
         assertex(conn.get()); // must be attached
         CriticalBlock block(sect);
         checkModify("CDistributedSuperFile::detach");
-        subfiles.kill();    
+        StringBuffer reason;
+        if (checkOwned(reason))
```

Comment:
Same comment as above. I can read it 'the reason of the check Owned'. Maybe the 'result' or similar would be better.